### ISSUE TITLE
SCT-5-WireframesThroughFigma

### ISSUE THREAD

**Issue by MostDeadDeveloper** [MostDeadDeveloper/student-concern-ticketing-system#5]
Let us start developing the initial wireframes!

A wireframe is by definition the following:

> A wireframe is a two-dimensional illustration of a page's interface that specifically focuses on space allocation and prioritization of content, functionalities available, and intended behaviors. For these reasons, wireframes typically do not include any styling, color, or graphics.

> In Simpler Terms, its a visualization of specific UI parts of our app. This will be a handy way for us to know exactly what a page should look.

For this issue, we need a wireframe for the following pages:
- Initial Dashboard (Admin)
- Login Page
- Ticket List Page ( Admin)

**Comment by Mica56** [comment]
Here's the figma link: https://www.figma.com/files/project/42739968/CSS-Project?fuid=1041388917585205359

btw, yung initial dashboard ba ay parang home page ganun? @MostDeadDeveloper

**Comment by MostDeadDeveloper** [comment]
Yup, for now keep it simple yung dashboard.

It just needs to list down existing tickets, with a specific section for
filtering.

**Comment by elcuenca** [comment]
Eyo @Mica56! Yung link ata is not working at my end or inup-date mo lang yung wireframes? 
![image](https://user-images.githubusercontent.com/60186871/144764112-fbac5190-280f-46b2-8f3a-39a93c9be5de.png)

**Comment by Mica56** [comment]
> Eyo @Mica56! Yung link ata is not working at my end or inup-date mo lang yung wireframes? ![image](https://user-images.githubusercontent.com/60186871/144764112-fbac5190-280f-46b2-8f3a-39a93c9be5de.png)

Slr, nakareceive ka ba on your email (yung elmercuenca74@gmail.com) ng invite? Need kasi nun in order to access. Paaccept nalang nung invite :>.

**Comment by elcuenca** [comment]
ohhhh wait sige hanapin ko na lang
